src/components/toplayer/_toplayer.scss

```diff
@@ -74,6 +74,29 @@ $includeHtml: false !default;
       }
     }
 
+    &--small-spaced {
+      @include sgBreakpoint(small-only) {
+        margin-top: gutter(4);
+        max-width: gutter(14);
+        margin-left: gutter(1);
+        margin-right: gutter(1);
+      }
+    }
+
+    &--splash-screen {
```

Comment:
it was named `--full-screen` but it's ALMOST full screen because of margins
is `--splash-scren` ok here?